presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -20,6 +20,10 @@ MULTIPOLYGON (((0 0, 4 0, 4 4, 0 4, 0 0), (1 1, 2 1, 2 2, 1 2, 1 1)), ((-1 -1, -
 
 GEOMETRYCOLLECTION (POINT(2 3), LINESTRING (2 3, 3 4))
 
+The set of GeoSpatial functions also includes functions to convert between geometries and Bing tiles.
+
+Bing tiles are described in https://msdn.microsoft.com/en-us/library/bb259689.aspx
```

Comment:
Remove this second sentence and make "Bing tiles" in the first sentence a hyperlink. Also, move this to the new "Bing Tiles" section.